libraries/chain/controller.cpp

```diff
@@ -1694,12 +1689,12 @@ struct controller_impl {
       // If we start at a block during or after the IF transition, we need to provide this information
       // at startup.
       // ---------------------------------------------------------------------------------------------
-      if (fork_db.fork_db_if_present()) {
+      if (auto in_use = fork_db.version_in_use(); in_use  == fork_database::in_use_t::both || in_use  == fork_database::in_use_t::savanna) {
```

Comment:
Maybe you can have a method `version_is_savanna()` and `version_is_legacy()` to avoid this long expression. But no need to change now.